config/sync/sparql_entity_storage.mapping.rdf_entity.rdf_graph.yml

```diff
@@ -26,7 +26,7 @@ base_fields_mapping:
       format: t_literal
   created:
     value:
-      predicate: 'http://purl.org/dc/terms/modified'
+      predicate: 'http://purl.org/dc/terms/issued'
```

Comment:
Nice catch. However, what happens if, at the time this is released, we already have have `rdf_graph`entities? Maybe we don't have in this moment, but we cannot anticipate at the release time.

Reply:
I have an update path to fix the values.